arteria/handlers/arteria_runfolder_handlers.py

```diff
@@ -0,0 +1,156 @@
+from aiohttp import web
+from pathlib import Path
+from arteria.models.state import State
+from arteria.models.config import Config
+from arteria.handlers.base import base_routes
+from arteria.models.runfolder_utils import Runfolder, list_runfolders
+
+
+routes = base_routes
+
+
+@routes.post("/runfolders/path/{runfolder}")
+async def post_runfolders(request):
+    """
+    When this is called with payload {"state": "STARTED"},
+    the state of the runfolder is set to STARTED
+    """
+    data = await request.post()
+    try:
+        runfolder_name = request.match_info['runfolder']
+        for runfolder_path in Config().get('monitored_directories'):
+            runfolder_cls = Runfolder(Path(runfolder_path) / runfolder_name)
+            state = data["state"]
+            if state not in State.__members__:
+                raise web.HTTPBadRequest(reason=f"The state '{state}' is not valid")
+
+            runfolder_cls.state = [s for s in State if s.name == state][0]
+
+        return web.json_response(status=200)
+    except Exception as e:
+        set_exceptions(e, kwargs={"state": data["state"]})
+
+
+@routes.get("/runfolders/path/{runfolder}")
+async def get_runfolders(request):
+    """
+    Returns some information about the runfolder as json
+    """
+    try:
+        runfolder_name = request.match_info['runfolder']
+        runfolders = [
+
+            serialize_runfolder_path(Runfolder(Path(runfolder_path) / runfolder_name), request)
+            for runfolder_path in Config().get('monitored_directories')
+        ]
+
+        return web.json_response(
+            data=runfolders,
+            status=200
+        )
+    except Exception as e:
+        set_exceptions(e)
+
+
+@routes.get("/runfolders/next")
+async def get_next_runfolder(request):
+    """
+    Finds unprocessed runfolder (state=ready) and then
+    returns some information about this runfolder.
+    """
+    try:
+        runfolder_cls = list_runfolders(

```

Comment:
This variable should be named something else, to emphasize that it's a list of runfolders. For instance `runfolders`.